packages/rich-text/src/test/__snapshots__/index.native.js.snap

```diff
@@ -0,0 +1,65 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<RichText/> Font Size renders component with style and font size 1`] = `
+"<!-- wp:paragraph {\\"style\\":{\\"color\\":{\\"text\\":\\"#fcb900\\"},\\"typography\\":{\\"fontSize\\":35.56}}} -->
+<p class=\\"has-text-color\\" style=\\"color:#fcb900;font-size:35.56px\\">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed imperdiet ut nibh vitae ornare. Sed auctor nec augue at blandit.</p>
+<!-- /wp:paragraph -->"
+`;
+
+exports[`<RichText/> Font Size should update the font size when style prop with font size property is provided 1`] = `
+<View>
+  <TextInput
+    accessibilityLabel="Text input. Empty"
+    activeFormats={Array []}
+    blockType={
+      Object {
+        "tag": "div",
+      }
+    }
+    disableEditingMenu={false}
+    fontFamily="serif"
+    fontSize={12}
+    isMultiline={false}
+    maxImagesWidth={200}
+    onBlur={[Function]}
+    onChange={[Function]}
+    onContentSizeChange={[Function]}
+    onFocus={[Function]}
+    onKeyDown={[Function]}
+    onPaste={[Function]}
+    onSelectionChange={[Function]}
+    placeholderTextColor="gray"
+    triggerKeyCodes={Array []}
+    value=""
+  />
+</View>
+`;
+
+exports[`<RichText/> Font Size should update the font size with decimals when style prop with font size property is provided 1`] = `
+<View>
+  <TextInput
+    accessibilityLabel="Text input. Empty"
+    activeFormats={Array []}
+    blockType={
+      Object {
+        "tag": "div",
+      }
+    }
+    disableEditingMenu={false}
+    fontFamily="serif"
+    fontSize={13}
```

Comment:
I'm wondering if the value should be rounded, as the provided font-size value is `12.56px` ([reference](https://github.com/WordPress/gutenberg/blob/60ca367fba5b2ea3d976151cb17746eec243de4f/packages/rich-text/src/test/index.native.js#L242)) 🤔 . I tested this behavior on the web version by setting the value manually via the block settings and the HTML conserved the decimals, @geriux do you know what's the expected behavior in this case?


<img width="600" alt="Screenshot 2022-01-14 at 14 24 47" src="https://user-images.githubusercontent.com/14905380/149522369-ef6ae935-d9ac-4424-a65a-959198c37c19.png">

**HTML output:**
```
<!-- wp:paragraph {"style":{"typography":{"fontSize":"12.56px"}}} -->
<p style="font-size:12.56px">Hello world!</p>
<!-- /wp:paragraph -->
```

Reply:
Currently, we are rounding the value yes. We could change this behavior when we add support to add custom font sizes. Thanks for double-checking!